assets/helpers/payPalExpressCheckout/payPalExpressCheckout.js

```diff
@@ -113,15 +116,43 @@ function setup(dispatch: Function, getState: () => CombinedState, callback: Func
         commit: true,
 
         // This function is called when user clicks the PayPal button.
-        payment: setupPayment(dispatch, getState()),
+
+        payment: () => {
```

Comment:
Since it is an inline arrow function which returns a Promise. I would extract all of it in a separate function to increase readability. The arrow-inline functions can be hard to read.

Reply:
In addition, since it was so hard to find the documentation, I would include the docs link in the comment above.

Reply:
Do you have the link to the documentation?

Reply:
The most accurate docs I think are from the Advance Js integration: [here](https://github.com/paypal/paypal-checkout/blob/master/docs/button.md#advanced-integration)

Reply:
Thanks